### PR TITLE
log successful telemetry attempts

### PR DESCRIPTION
**Pull request by jakedt**

### CODE REVIEW

File: internal/telemetry/reporter.go [hunk 1]
```diff
@@ -194,6 +194,10 @@ func RemoteReporter(
 						Stringer("next", nextPush).
 						Msg("failed to push telemetry metric")
 				} else {
+					log.Info().
```
Comment (jzelinskie) [inline internal/telemetry/reporter.go:197]: perhaps this should be debug?

Comment (josephschorr) [reply internal/telemetry/reporter.go:197]: +1 to this being debug

**Review by ecordell — APPROVED** [review]
LGTM

**Review by josephschorr — APPROVED** [review]
LGTM

**Pull request merged**